src/components/themed/TransactionListRow.tsx

```diff
@@ -115,13 +117,22 @@ export function TransactionListRow(props: Props) {
     })
   })
 
+  const handleLongPress = useHandler(() => {
+    const url = sprintf(currencyInfo.transactionExplorer, transaction.txid)
+    const shareOptions = {
+      url
+    }
+    Share.open(shareOptions).catch(e => console.log(e))
```

Comment:
We generally use `.catch(showError)` instead of logging to console, unless we _want_ to hide the error for some reason. Since this is a case of direct user interaction, we really should show the error so they know why their long tap didn't work.

Reply:
Changed to `.catch(e => showError(e))`